tests/test_aaa_authorization_service.rb

```diff
@@ -92,6 +95,14 @@ def tacacs_groups
     %w(tac_group bxb100 sjc200 rtp10)
   end
 
+  def valid_cfg?(cfg)
```

Comment:
@mikewiebe I found a way to verify the server without using ping.  It involves [using the `test` command to see if the server is reachable and a tacacs server](http://www.cisco.com/c/en/us/td/docs/switches/datacenter/sw/4_1/nx-os/security/configuration/guide/sec_nx-os-cfg/sec_tacacsplus.html#wpxref30015)